CONTRIBUTING.md

```diff
@@ -124,3 +124,73 @@ what a commit is about.
 
 When you create a pull request, please
 [make your PR editable](https://github.com/blog/2247-improving-collaboration-with-forks).
+
+## Rebasing
+
+If other PRs have been merged during the time between your initial PR creation
+and final approval, it may be required that you rebase your changes against the
+latest `main` branch.
+
+There are potential pitfalls here if you follow the suggestions from `git`,
+which could leave your branch in an unrecoverable mess,
+and you having to start over with a new branch and new PR.
+
+The procedure below is tried and tested, and will help you avoid frustration.
+
+To rebase a feature branch to the latest `main`:
+
+1. Make sure that your local copy of the repository has the most up-to-date
+   revisions of `main` (this is important, otherwise you may end up rebasing to
+   an older base point):
+   ```bash
+   git switch main
+   git pull
+   ```
+1. Switch to the (feature) branch to be rebased and make sure your copy is up to
+   date:
+   ```bash
+   git switch feature/something-cool
+   git pull
+   ```
+1. Consider taking a copy of the folder tree at this stage; this may help when
```

Comment:
TBH, I don’t like that every instruction list item starts with 1. The entire point of markdown is that it’s possible to understand even when reading plain text.

Reply:
OK.  The advantage of `1.` is that it minimizes diffs if an item is added within the list, in turn helping with merging (if required).  I think you are suggesting that Markdown should not have this feature at all :)